src/views/CashlinkManage.vue

```diff
@@ -17,38 +17,34 @@
-                    <svg width="28" height="24" viewBox="0 0 28 24" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
-                        <g opacity=".6">
-                            <path d="M13.9047 11.3644C14.0384 11.4981 14.1721 11.4981 14.3058 11.3644L27.1408 2.80767C27.4082 2.67397 27.5419 2.40658 27.4082 2.13918C27.1408 0.935891 26.0713 0 24.7343 0H2.67397C1.33699 0 0.267397 0.935891 0 2.13918C0 2.40658 0.133699 2.67397 0.401096 2.94137L13.9047 11.3644Z"/>
-                            <path d="M15.3754 14.4395C14.5732 14.9743 13.6373 14.9743 12.8351 14.4395L1.06959 6.95235C0.668494 6.81865 0.267398 6.95235 0.133699 7.21975C0 7.35345 0 7.48715 0 7.62085V20.7233C0 22.194 1.20329 23.3973 2.67398 23.3973H24.7343C26.205 23.3973 27.4082 22.194 27.4082 20.7233V7.62085C27.4082 7.21975 27.1408 6.95235 26.7398 6.95235C26.6061 6.95235 26.4724 6.95235 26.3387 7.08605L15.3754 14.4395Z"/>
-                        </g>
-                    </svg>
-                </a>
-                <a class="nq-button-s social-share" :href="whatsapp" target="_blank">
-                    <svg width="28" height="28" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
-                        <path opacity=".6" d="M14 0C6.3 0 0 6.235 0 14c0 2.823.817 5.647 2.45 8L.817 26.706c-.117.235 0 .47.116.588.117.118.35.235.584.118l5.016-1.53C13.067 30 21.7 28 25.783 21.412c1.4-2.235 2.1-4.824 2.1-7.412C28 6.235 21.7 0 14 0zm8.4 19.647c-.817 1.53-2.333 2.47-4.083 2.47-1.4-.117-2.8-.588-3.967-1.176-2.8-1.294-5.25-3.412-7-6.117-2.217-2.942-2.333-5.765-.233-8.118.583-.47 1.4-.706 2.216-.588.584.117 1.167.47 1.4 1.058l.467 1.06c.35.823.7 1.646.7 1.764.233.353.233.823 0 1.177-.35.588-.7 1.176-1.167 1.646a10.68 10.68 0 0 0 2.1 2.471c.934.824 1.984 1.53 3.15 1.883.35-.471.934-1.177 1.167-1.53a1.175 1.175 0 0 1 1.517-.47c.466.117 2.916 1.411 2.916 1.411.35.118.584.353.817.588.467.942.467 1.765 0 2.471z"/>
-                    </svg>
-                </a>
+                <template v-else>
+                    <a class="nq-button-s social-share telegram" :href="telegram" target="_blank">
+                        <svg width="29" height="25" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+                            <path opacity=".6" d="M28.537.528c-.103-.308-.224-.397-.416-.472-.42-.164-1.124.084-1.124.084S1.943 9.496.516 10.534c-.308.225-.416.351-.467.505-.247.74.523 1.062.523 1.062l6.458 2.185s.023.005.056.005l3.024 8.7s.275.585.61.777c.01.01.024.014.033.023l.028.014c.038.019.08.028.131.028.005 0 .01.005.014.005h.005c.252 0 .658-.22 1.311-.903a56.694 56.694 0 013.392-3.227c2.221 1.595 4.61 3.354 5.642 4.275.518.463.952.538 1.306.524.98-.038 1.251-1.155 1.251-1.155s4.573-19.09 4.722-21.65c.014-.252.038-.411.038-.584.004-.239-.019-.477-.056-.59zM8.332 13.758l.004-.132c3.463-2.268 13.887-9.089 14.568-9.35.122-.038.21.004.187.093-.308 1.123-11.866 11.797-11.866 11.797s-.042.056-.07.122l-.014-.01-.397 4.398-2.412-6.919z"/>
+                        </svg>
```

Comment:
One decimal place can be shaved off all coordinates in this SVG without any visual change:

```html
<svg width="29" height="25" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
    <path opacity=".6" d="M28.54.53c-.1-.31-.23-.4-.42-.47C27.7-.11 27 .14 27 .14S1.94 9.5.52 10.54c-.31.22-.42.35-.47.5-.25.74.52 1.06.52 1.06l6.46 2.19h.06L10.1 23s.28.59.61.78l.03.02.03.02.13.02h.02c.25 0 .66-.21 1.31-.9a56.7 56.7 0 013.4-3.22 90.48 90.48 0 015.64 4.27c.51.47.95.54 1.3.53.98-.04 1.25-1.16 1.25-1.16S28.41 4.26 28.56 1.7c0-.25.03-.4.03-.58 0-.24-.02-.48-.05-.6zM8.34 13.76v-.13c3.46-2.27 13.88-9.1 14.56-9.35.13-.04.21 0 .2.09-.32 1.12-11.88 11.8-11.88 11.8l-.06.12-.02-.01-.4 4.4-2.4-6.92z"/>
</svg>
```